Misc/NEWS.d/next/Core and Builtins/2017-09-26-13-03-16.bpo-31588.wT9Iy7.rst

```diff
@@ -0,0 +1,2 @@
+Prevent raising a SystemError in class creation in case of a metaclass with
```

Comment:
It is not clear what is the new behavior. Is a class created successfully?

Document that a TypeError is now raised instead of a SystemError.